  @Test
  public void test21() throws Exception {
    helper2("fValue", "fOrdinal");
  }

